modules/js-and-css/speculation-rules/helper.php

```diff
@@ -67,7 +57,7 @@ static function ( $exclude_path ) use ( $prefixer ) {
 				'and' => array(
 					// Prerender any URLs within the same site.
 					array(
-						'href_matches' => $prefixer->prefix_path_pattern( '/*\\?*' ),
+						'href_matches' => $prefixer->prefix_path_pattern( '/*' ),
```

Comment:
@domenic Same here, can you confirm this is safe to drop?

Reply:
Confirmed!